modules/arima/2readsbamcombiner/meta.yml

```
@@ -0,0 +1,46 @@
+name: arima_2readsbamcombiner
+
+description: Combin two paired single read bam files into one sorted bam
+keywords:
+  - samtools
+  - combine
+  - sort
+  - arima
+tools:
+  - arima:
+      description: Combin paired subgke read bam files into one sorted bam using Samtools and Perl script
+      homepage: https://arimagenomics.com/
+      documentation: https://usermanual.wiki/Document/ArimaMappingUserGuideA160156v01.424413804/view
```

Do you know if this file was uploaded by Arima themselves ? If you're not sure, better to use the version that is on their GitHub
```suggestion
      documentation: https://github.com/ArimaGenomics/mapping_pipeline/blob/master/Arima_Mapping_UserGuide_A160156_v02.pdf
```